docs/examples/private_attributes.py

```diff
@@ -0,0 +1,18 @@
+from datetime import datetime
+from random import randint
+
+from pydantic import BaseModel, PrivateAttr
+
+
+class TimeAwareModel(BaseModel):
+    _processed_at: datetime = PrivateAttr(default_factory=datetime.now)
+    _secret_value: str = PrivateAttr()
+
+    def __init__(self, **data):
+        super().__init__(**data)
+        self._secret_value = randint(1, 5)
```

Comment:
maybe add a commit that this could also be done with `default_factory`

Reply:
you mean comment?

Reply:
_yes_